scenes/game/game.gd

```diff
@@ -3130,6 +3151,11 @@ func _update_buttons():
 	action_menu.set_choices(current_instruction_text, button_choices, ultra_force_toggle, instructions_number_picker_min, instructions_number_picker_max, ex_discard_order_toggle)
 	current_action_menu_choices = button_choices
 
+func _choice_text_without_tags(choice_text):
+	var regex = RegEx.new()
```

Comment:
can you save this regex somewhere that it doesn't create a new one every time it builds the choice strings?